SliderButton/SliderButtonBehaviorDecorator.js

```diff
@@ -0,0 +1,48 @@
+import platform from '@enact/core/platform';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {findDOMNode} from 'react-dom';
+
+// Adds agate-specific SliderButton behaviors
+// * aria-valuetext handling
```

Comment:
The comment format is strange to me. Could you check it?

Reply:
I just copied from `SliderBehaviorDecorator`.